developer_manual/basics/storage/filesystem.rst

```diff
@@ -24,7 +24,6 @@ Access to the filesystem is provided by the ``IRootFolder`` which can be injecte
 From the root folder you can either access a user's home folder or access a file or folder by its absolute path.
 
 .. code-block:: php
-    <?php
```

Comment:
these block were absent in documentation due to parsing error, with these changes it is ok